src/CG-DNA/pair_oxdna2_coaxstk.cpp

```diff
@@ -556,7 +559,7 @@ void PairOxdna2Coaxstk::coeff(int narg, char **arg)
 {
   int count;
 
-  if (narg != 21) error->all(FLERR,"Incorrect args for pair coefficients in oxdna2/coaxstk");
+  if (narg != 3 && narg != 21) error->all(FLERR,"Incorrect args for pair coefficients in oxdna2/coaxstk");
```

Comment:
Should this not be `OR` in the condition? Throw error if narg is not 3 and not 21.